packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -184,7 +186,8 @@ export class DetailsRow extends React.Component<IDetailsRowProps, IDetailsRowSta
         data-automationid='DetailsRow'
         style={ { minWidth: viewport ? viewport.width : 0 } }
         aria-selected={ isSelected }
-        >
+        onContextMenu={ (ev) => { this._onItemContextMenu(this.props.item, this.props.itemIndex, ev); } }
```

Comment:
This still creates a new function for every row. Not sure how this will affect performance of the `DetailsList`. Could we have a single event handler further up in the tree and identify the element the event was triggered for there? 

@dzearing how do you usually handle this?